pkg/storage/engine/bench_test.go

```diff
@@ -533,12 +535,18 @@ func runMVCCDeleteRange(emk engineMaker, valueBytes int, b *testing.B) {
 	b.StopTimer()
 	b.ResetTimer()
 
+	locDirty := dir + "_dirty"
+
 	for i := 0; i < b.N; i++ {
-		locDirty := dir + "_dirty"
 		if err := os.RemoveAll(locDirty); err != nil {
 			b.Fatal(err)
 		}
 		if err := copyDir(dir, locDirty); err != nil {
+			if os.IsNotExist(err) {
```

Comment:
Hmm, this looks creepy. Can't you close `eng` before getting here? I assume that's why you get errors here in the first place.